CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
 # Changelog
 
+### 1.9.0
+
+OVERVIEW:
+
+Release 1.9.0 includes:
+* Improved Prometheus metrics.
+* Support for Rate Limiting, JWT, IngressMTLS, EgressMTLS via the policy resource.
+* Support for NGINX Service Mesh.
```

Comment:
Perhaps?
* Support for the latest Ingress resource features and the IngressClass resource.